tests/compute/continue-stmt.slang.expected.txt

```diff
@@ -0,0 +1,4 @@
+0
```

Comment:
According to C++'s continue semantics in a do-while statement, the output of this test case should be 
0
FFFFFFFF
FFFFFFFF
FFFFFFFF

Reply:
Okay, I’ll fix it.

`do/while` loops are terrible anyway, so I’m not ashamed to not remember how they work. :)

Reply:
This should be fixed now, but I reworked the test case so that the loop actually runs instead of terminating right away.

Thanks for making this catch, since it actually exposed a more serious issue around `continue` that also affects `for` loops (which people actually use, unlike `do/while`).